pyproject.toml

```diff
@@ -22,13 +23,20 @@ dependencies = [
     "numpy",
```

Comment:
Might want to pin numpy in advance of their 2.0 release

Reply:
Good point — are you thinking "numpy<2.0" or pinning to a specific version?

Reply:
<2.0 should be sufficient

Reply:
and done!